perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -58,6 +58,18 @@
 	 */
 	void deleteMember(PerunSession sess, Member member) throws InternalErrorException, MemberNotExistsException, PrivilegeException, MemberAlreadyRemovedException;
 
+	/**
+	 *  Delete given members.
```

Comment:
Two spaces instead of one. And you should probably inform about ability of this method to removing members from different virtual organizations at once.

Reply:
I have updated the Javadocs.